File: Ginger/src/net/pic4pic/ginger/entities/AppStoreType.java
package net.pic4pic.ginger.entities;

public enum AppStoreType implements IntegerEnum {
	
	Unknown(0),
    GooglePlay(1);
    
    private final int value;
	
	private AppStoreType(int value) {
		this.value = value;
	}

	public int getIntValue() {
		return this.value;
	}
}

File: Pic4Pic/Pic4Pic.ObjectModel/Entities/AppStoreType.cs
using System;
using System.Runtime.Serialization;

namespace Pic4Pic.ObjectModel
{
    [DataContract()]
    public enum AppStoreType
    {
        [EnumMember]
        Unknown = 0,

        [EnumMember]
        GooglePlay = 1,
    }
}
